CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,81 @@
+# Up-For-Grabs Code of Conduct
+
+This document outlines how contributors to the Up-For-Grabs project
+should interact with each other, and the process for resolving issues.
+
+Projects listed on Up-For-Grabs are not covered by this code of conduct.
```

Comment:
I think this is worded nicely and clearly. 👍

Reply:
Only because "should" is used in place of "must". In actual reality it codifies behaviour, and requires pledging allegiance to it to participate. How rosy it sounds is beside the point of how authoritarian that is.
Not that undermining meritocracy (which is what it is actually meant to do) isn't riding fundamentally on signalling virtue.
How you caught "deemed" and still believed in this I don't understand.